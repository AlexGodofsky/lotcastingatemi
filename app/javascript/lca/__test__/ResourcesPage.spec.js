import React from 'react'
import ResourcesPage from '../components/pages/resourcesPage.jsx'
import renderer from 'react-test-renderer'

test('ResourcesPage renders at all', () => {
  const component = renderer.create(
    <ResourcesPage />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
