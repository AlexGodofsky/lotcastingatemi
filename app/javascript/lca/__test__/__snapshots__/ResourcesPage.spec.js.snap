// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResourcesPage renders at all 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation2-6 MuiPaper-rounded-3 BlockPaper-root-1"
>
  <h1
    className="MuiTypography-root-29 MuiTypography-headline-34 MuiTypography-gutterBottom-46"
  >
    Resources
  </h1>
  <p
    className="MuiTypography-root-29 MuiTypography-body1-38 MuiTypography-paragraph-47"
  >
    <a
      href="https://www.reddit.com/r/exalted/comments/4yby39/madletters_charm_cascades_version_3_including/"
    >
      Charm Cascades and other resources
    </a>
      by MadLetter
  </p>
  <p
    className="MuiTypography-root-29 MuiTypography-body1-38 MuiTypography-paragraph-47"
  >
    <a
      href="http://forum.theonyxpath.com/forum/main-category/exalted/766048-ex3-i-made-a-1-page-combat-flowchart"
    >
      Combat flowchart
    </a>
      by GivenFlesh
  </p>
  <p
    className="MuiTypography-root-29 MuiTypography-body1-38 MuiTypography-paragraph-47"
  >
    <a
      href="http://forum.theonyxpath.com/forum/main-category/exalted/1180122-ex3-social-system-cheat-sheet-and-flowchart"
    >
      Social system cheat sheet
    </a>
      by Redthorn
  </p>
  <p
    className="MuiTypography-root-29 MuiTypography-body1-38 MuiTypography-paragraph-47"
  >
    <a
      href="http://forum.theonyxpath.com/forum/main-category/exalted/65025-exalted-3rd-edition-useful-links-and-topics"
    >
      'Useful Links' thread on the official Exalted forums
    </a>
  </p>
</div>
`;
